ts/receiver/closedGroups.ts

```diff
@@ -885,6 +886,40 @@ export async function createClosedGroup(groupName: string, members: Array<string
   convo.updateLastMessage();
 
   // Send a closed group update message to all members individually
+  const groupInviteResults = await sendToGroupMembers(
+    listOfMembers,
+    groupPublicKey,
+    groupName,
+    admins,
+    encryptionKeyPair,
+    dbMessage
+  );
+
+  const allInvitesSent = groupInviteResults.every(result => result === true);
+  if (!allInvitesSent) {
+    window.confirmationDialog({
+      title: 'Group Invite Failed',
+      message: 'Unable to successfully invite all group members',
```

Comment:
those values need to be internationalised => see window.i18n for usage. (you have to set the key and message in _locales/en only